app/utils/SummaryListRowHelper.scala

```diff
@@ -44,6 +44,9 @@ class SummaryListRowHelper(implicit messages: Messages) {
 
   protected def formatAsText[T](answer: T): Content = s"$answer".toText
 
+  protected def formatAsCoordinates(answer: GNSSType): Content =
+    Seq(answer.latitude, answer.longitude).mkString(",").toText
```

Comment:
@prashanthmamidi do you know where this requirement came from? Or is it just a guess? I can't see anything on the ticket. In the IE015 we formatted it like [this](https://github.com/hmrc/ctc-departure-route-details-frontend/blob/44b5baddc9979c765f9b7d98ed483924a7442c58/app/models/Coordinates.scala#L25).

Might be worth checking with Gavin and Nige

Reply:
@jackheslop96  it's just a guess - I think it matches with what we done in IE015 - latitude, longitude.
Will check with Gavin and see if there's any specific format we need to display

Reply:
Updated as (latitude, longitude) format